lib.php

```diff
@@ -207,3 +198,29 @@ function tool_crawler_extend_navigation_course($navigation, $course, $coursecont
         }
     }
 }
+/**
+ * Add a batch of ad hoc crawl tasks
+ * @param integer $limitadhoctasks the limit of concurrent adhoc crawl tasks
+ * @param boolean $verbose show verbose feedback
+ */
+function tool_crawler_add_adhoc_task($limitadhoctasks, $verbose = false) {
```

Comment:
Is there any reason this is in local lib and not just a method in the task class?

Reply:
Have moved this over to the task class, it's not used anywhere else